docs/decisions/0002-separate-exam-store.rst

```diff
@@ -0,0 +1,55 @@
+2. Use independent exam redux store
```

Comment:
this sounds like it's based on another ADR. If so, can we reference that?

Reply:
Removed decision after refactoring